actions/icon_overlay.rb

```diff
@@ -39,31 +28,33 @@ def self.caption
         version = ENV['ICON_OVERLAY_APP_VERSION']
         commit = `git rev-parse HEAD | head -1`.strip
         time = Time.new
-        scheme = ENV['ICON_OVERLAY_TITLE'] || ENV['GYM_SCHEME'] || "scheme"
-        ["#{version} #{scheme}",
-         "#{time.day}/#{time.month} #{time.hour}:#{time.min}",
-         "#{commit[0..7]}"
+        scheme = ENV['ICON_OVERLAY_TITLE'] || ENV['GYM_SCHEME'] || 'scheme'
+        [
+          "#{version} #{scheme}",
+          "#{time.day}/#{time.month} #{time.hour}:#{time.min}",
+          commit[0..7].to_s
         ].join("\n")
       end
 
-      def self.convert(source_path, destination_path)
-        if not File.exists? source_path
+      def self.convert(source_path, destination_path) # rubocop:disable Metrics/AbcSize
+        if !File.exist? source_path
```

Comment:
I think there's a way we can make this statement simpler but it's not leaping to me

Reply:
I think this okay for now. I think given how much has changed I'm eager to do some regression testing, not sure where to start though! guess we need to just step through each individual thing the master fastfile can do and just check it still works somehow